catalyst/dl/callbacks/checkpoint.py

```diff
@@ -98,7 +98,7 @@ def __init__(
         self.resume = resume
         self.resume_dir = resume_dir
         self.top_best_metrics = []
-
+        self.all_metrics = []
```

Comment:
why `all_metrics`? are they... `epoch_metrics`?

Reply:
Yeah, that's better. Named it relative to "best" ones.

Reply:
But there is epoch metrics local variable in `on_epoch_end` method.

Reply:
Maybe `epochs_metrics`? To suppose that we have all epochs records on main metric tracked.

Reply:
`epochs_metrics` looks good